WcaOnRails/config/locales/fr.yml

```diff
@@ -1267,9 +1267,9 @@ fr:
       website: Site internet
       #original_hash: 680567f
       organizer_plural:
-        other: Organisateurs
-        one: Organisateur
         zero: Organisateur
+        one: Organisateur
```

Comment:
@jonatanklosko do you have an idea on how to prevent these reordering from happening?
Is it possible to have the pluralization arrays sorted before exporting the translation?